docs/Deploying/Configuration.md

```diff
@@ -2118,6 +2118,21 @@ Maximum age of a vulnerability (a CVE) to be considered "recent". The age is cal
        recent_vulnerability_max_age: 48h
   ```
 
+### disable_win_os_vulnerabilities 
+
+If using osquery 5.4 or later, Fleet by default will fetch and store all applied Windows updates and use that for detecting Windows
```

Comment:
Hey @Desmi-Dizney ! When you get a chance could you please :eyes: this doc entry and tell me if it makes sense? I'm looking to merge this PR today if possible - Thanks!

Reply:
I think @chris-mcgillicuddy is in charge of this now